client/src/main/java/io/split/engine/matchers/Transformers.java

```diff
@@ -58,6 +60,24 @@ public static Long asDateHourMinute(Object obj) {
         return c.getTimeInMillis();
     }
 
+    public static Boolean asBoolean(Object obj) {
+        if (obj == null) {
+            return null;
+        }
+
+        if (obj instanceof Boolean) {
+            return (Boolean) obj;
+        }
+
+        if (obj instanceof String) {
+            if (Sets.newHashSet("true", "false").contains(((String) obj).toLowerCase())) {
```

Comment:
have a static set so its not initialized every time?, same with the (String obj)